backend/src/test/repository/EntityIntegration.test.ts

```diff
@@ -46,7 +46,7 @@ describe('Integration tests for all entities', function () {
       fail('Should have failed');
     } catch (e) {
       expect(e.message).toBe(
-        'null value in column "userId" violates not-null constraint'
+        'null value in column "userId" of relation "promotion" violates not-null constraint'
```

Comment:
Same comment here

Reply:
Sorry 😅  which comment?

Reply:
Oh I realize github hides comments when there's a lot, just above I think you need to click load more
![image](https://user-images.githubusercontent.com/49849754/102852332-dc1ccc00-43d2-11eb-8c60-c281afcc4698.png)